data/json/item_actions.json

```diff
@@ -271,7 +271,7 @@
   },
   {
     "type": "item_action",
-    "id": "ARROW_FLAMABLE",
```

Comment:
Same here, make sure that all the places using this have a changed spelling (or probably better, don't change it).

Reply:
I did replace it everywhere it was found.